cmd/buildah/common.go

```diff
@@ -108,3 +111,50 @@ func getDateAndDigestAndSize(image storage.Image, store storage.Store) (time.Tim
 	}
 	return created, manifestDigest, imgSize, err
 }
+
+// ContextFromGlobalOptions returns a SystemContext populated with values
+// per the input parameters provided by the caller for the use in authentication.
+func ContextFromGlobalOptions(c *cli.Context, flagPrefix string) (*types.SystemContext, error) {
```

Comment:
Ok, I was able to drop the param.  I thought go had barked at me prior if I didn't export this, but I've renamed as you suggested and it appears to be happy.  Thanks!